app/Http/Requests/DemoTestRequest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class DemoTestRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'objects' => 'required|array|max:2000',
```

Comment:
Not implemented
7. Validation must be added for the endpoint:
	If request conatins at least one object that present in db and it's inactive then validation error 422 must be thrown

Reply:
implemente